.github/workflows/_system_test.yml

```diff
@@ -50,4 +50,4 @@ jobs:
         run: blueapi -c ${{ github.workspace }}/tests/system_tests/config.yaml serve &
 
       - name: Run tests
-        run: tox -e system-test
+        run: uv run --frozen tox -e system-test
```

Comment:
Hmm, maybe `--frozen` does make sense here, but maybe `--locked` would make even more sense? What do you reckon?

Reply:
`--locked` fails if any packages are updatable so would fail pretty much every run, `--frozen` makes it use the versions in the lock file.